### DISCUSSION TITLE
dutch translation [May 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2206] [Discord Threads]
> 🙋‍♂️ **TsjipTsjip**  
## Where can I find these keys? I'm working on a translation to Dutch and need context.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/955747155929346068/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### the keys are used in frontend code.  
#  
> 👨‍🚀 **wingkwong**  
#### just FYI. dutch translation should be on column I.  
#  
> 🙋‍♂️ **TsjipTsjip**  
#### Yeah, I saw, but it could really use some work.  
#  
> 🙋‍♂️ **TsjipTsjip**  
#### And I understand the keys bit, I just can't come up with a good translation without knowing where they are being used.  
#  
> 👨‍🚀 **wingkwong**  
#### which ones u r looking for ?  
#  
> 🙋‍♂️ **TsjipTsjip**  
#### Specifically "Inflection - ..."  
#  
> 🙋‍♂️ **TsjipTsjip**  
#### Not sure what that means or where that is (I haven't used NocoDB for that long either, might just be something I haven't encountered.)  
#  
> 👨‍🚀 **wingkwong**  
#### inflection .. can be found under SSL & Advanced parameters when creating a project  
![image0](https://cdn.discordapp.com/attachments/955747156621393962/955749932269207572/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### inflection s like camelize, pluralize, underscore etc  
#  
> 💁 **o1lab**  
#### dstala@discord can help on the translation bit  
#  
> 🙋‍♂️ **TsjipTsjip**  
#### I've opened an issue with my version of the spreadsheet. Still a bit unsure about the "Inflexion" bit, as I don't really understand what is meant there. All the rest should be fine, however I am open for corrections.   
####    
#### Thanks for the assistance!  
#  
> 👨‍🦲 **dstala**  
#### TsjipTsjip@discord Will merge recommendations  
#  
> 👨‍🦲 **dstala**  
#### https://github.com/nocodb/nocodb/commit/2a7f0d1e03bb847299d801d2c99fadf29fb9ef44   
####    
#### Merged onto Develop  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/955747156621393962)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
